src/models/Movie/parseMovieData.ts

```diff
@@ -15,22 +15,33 @@ export function parseMovieSummary(entry: MovieContentfulEntry): MovieSummary {
 
 // Function to parse detailed movie data (MovieModel)
 export function parseMovie(entry: MovieContentfulEntry): Movie {
+	const { fields } = entry;
+
+	return {
+		...parseMovieFields(entry),
+		genres: (fields.genres as GenreContentfulEntry[]).map((genre: GenreContentfulEntry) => ({
+			id: genre.sys.id,
+			title: genre.fields.title,
+		})),
+	};
+}
+
+//Helper function to parse the movie data
+function parseMovieFields(entry: MovieContentfulEntry): Omit<Movie, 'genres'> {
 	const summary = parseMovieSummary(entry);
 	const { fields } = entry;
 
 	return {
 		...summary,
-		summary: fields.summary,
-		duration: fields.duration,
-		directors: fields.directors,
-		mainActors: fields.mainActors,
-		genres: ((fields.genres as Array<GenreContentfulEntry>) || [])?.map(
-			(genre) => genre.fields?.title
-		),
-		datePublished: fields.datePublished,
-		ratingValue: fields.ratingValue,
-		bestRating: fields.bestRating,
-		worstRating: fields.worstRating,
-		writers: fields.writers,
+		summary: fields.summary || '',
+		duration: fields.duration || '',
+		directors: fields.directors || [],
+		mainActors: fields.mainActors || [],
+		datePublished: fields.datePublished || '',
+		rating: fields.rating || '',
+		ratingValue: fields.ratingValue || 0,
+		bestRating: fields.bestRating || 0,
+		worstRating: fields.worstRating || 0,
+		writers: fields.writers || [],
```

Comment:
Do we still need those fallbacks? The types issue got solved in the last merge.

If we still need them, this is breaking the endpoint `GET /movies/titles`